actions/subscribers_test.go

```diff
@@ -0,0 +1,109 @@
+package actions_test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/mailbadger/app/entities/params"
+	"github.com/mailbadger/app/storage"
+)
+
+func TestSubscribers(t *testing.T) {
+	s := storage.New("sqlite3", ":memory:")
+
+	e := setup(t, s)
+	auth, err := createAuthenticatedExpect(e, s)
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
+	}
+
+	e.POST("/api/subscribers").WithForm(params.PostSubscriber{Name: "", Email: "", Metadata: map[string]string{"": ""}, SegmentIDs: nil}).
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	// test binding on post subscriber
+	auth.POST("/api/subscribers").WithForm(params.PostSubscriber{Name: "", Email: "", Metadata: map[string]string{"": ""}, SegmentIDs: nil}).
+		Expect().
+		Status(http.StatusBadRequest).JSON().Object().
+		ValueEqual("message", "Invalid parameters, please try again").
+		ValueEqual("errors", map[string]string{"email": "This field is required"})
+
+	/*// test validator for params
+	auth.POST("/api/subscribers").WithForm(params.PostSubscriber{Name: "Djale", Email: "sda", Metadata: map[string]string{"asd asd": "bla bla"}, SegmentIDs: nil}).
+		Expect().
+		Status(http.StatusBadRequest).JSON().Object().
+		ValueEqual("message", "Invalid parameters, please try again").
+		ValueEqual("errors", map[string]string{"email": "Invalid email format", "metadata": "Must consist only of alphanumeric and hyphen characters"})
+	*/
+	// test post subscriber
+	auth.POST("/api/subscribers").WithForm(params.PostSubscriber{Name: "Djale", Email: "djale@email.com", Metadata: map[string]string{"test": "test"}, SegmentIDs: nil}).
+		Expect().
+		Status(http.StatusCreated)
+
+	// test post subscriber ( insert more then 1 sub)
+	auth.POST("/api/subscribers").WithForm(params.PostSubscriber{Name: "Foo", Email: "foo@email.com", Metadata: map[string]string{"test": "test"}, SegmentIDs: nil}).
+		Expect().
+		Status(http.StatusCreated)
+
+	// test get subscribers length 2
+	auth.GET("/api/subscribers").
+		Expect().
+		Status(http.StatusOK).JSON().Object().Value("collection").Array().Length().Equal(2)
```

Comment:
Format this file a bit better, also what should we do regarding the commented out code?